### ISSUE TITLE
toil-rnaseq, toil.batchSystems.abstractBatchSystem.InsufficientSystemResources (Disk)

### ISSUE THREAD

**Issue by hmkim** [DataBiosphere/toil#2505]
$ toil-rnaseq run ./jobStore

> INFO:toil.leader:Issued job 'Job' K/3/jobwKfS1P with job batch system ID: 10 and cores: 0, disk: 1.0 M, and memory: 32.0 M
> INFO:toil.leader:Job ended successfully: 'Job' K/3/jobwKfS1P
> INFO:toil.leader:Issued job 'EncapsulatedJob' z/t/job85EqgN with job batch system ID: 11 and cores: 0, disk: 1.0 M, and memory: 32.0 M
> INFO:toil.leader:Issued job 'EncapsulatedJob' q/B/joboXRNOc with job batch system ID: 12 and cores: 0, disk: 1.0 M, and memory: 32.0 M
> INFO:toil.leader:Issued job 'EncapsulatedJob' 1/U/jobAqDIBb with job batch system ID: 13 and cores: 0, disk: 1.0 M, and memory: 32.0 M
> INFO:toil.leader:Issued job 'EncapsulatedJob' p/1/job4wGvda with job batch system ID: 14 and cores: 0, disk: 1.0 M, and memory: 32.0 M
> INFO:toil.leader:Job ended successfully: 'EncapsulatedJob' z/t/job85EqgN
> INFO:toil.leader:Issued job 'run_fastqc' p/P/jobPsTQT_ with job batch system ID: 15 and cores: 2, disk: 2.0 G, and memory: 2.0 G
> INFO:toil.leader:Job ended successfully: 'EncapsulatedJob' q/B/joboXRNOc
> INFO:toil.leader:Issued job 'run_kallisto' 2/G/jobWqQxUR with job batch system ID: 16 and cores: 16, disk: 2.0 G, and memory: 2.0 G
> INFO:toil.leader:Job ended successfully: 'EncapsulatedJob' 1/U/jobAqDIBb
> INFO:toil.leader:Issued job 'run_hera' n/e/jobyUPGTz with job batch system ID: 17 and cores: 128, disk: 2.0 G, and memory: 2.0 G
> INFO:toil.leader:Job ended successfully: 'EncapsulatedJob' p/1/job4wGvda
> 
> Traceback (most recent call last):
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/bin/toil-rnaseq", line 9, in <module>
>     load_entry_point('toil-rnaseq==4.1.0', 'console_scripts', 'toil-rnaseq')()
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil_rnaseq/toil_rnaseq.py", line 214, in main
>     toil.start(Job.wrapJobFn(map_job, workflow, samples, config))
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil/common.py", line 784, in start
>     return self._runMainLoop(rootJobGraph)
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil/common.py", line 1059, in _runMainLoop
>     jobCache=self._jobCache).run()
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil/leader.py", line 208, in run
>     self.innerLoop()
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil/leader.py", line 515, in innerLoop
>     self._processReadyJobs()
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil/leader.py", line 439, in _processReadyJobs
>     self._processReadyJob(jobGraph, resultStatus)
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil/leader.py", line 409, in _processReadyJob
>     self._runJobSuccessors(jobGraph)
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil/leader.py", line 327, in _runJobSuccessors
>     self.issueJobs(successors)
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil/leader.py", line 608, in issueJobs
>     self.issueJob(job)
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil/leader.py", line 587, in issueJob
>     jobBatchSystemID = self.batchSystem.issueBatchJob(jobNode)
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil/batchSystems/singleMachine.py", line 236, in issueBatchJob
>     self.checkResourceRequest(jobNode.memory, cores, jobNode.disk)
>   File "/BiO/BioPeople/brandon/project_exp/toil-rnaseq/local/lib/python2.7/site-packages/toil/batchSystems/abstractBatchSystem.py", line 251, in checkResourceRequest
>     raise InsufficientSystemResources('disk', disk, self.maxDisk)
> toil.batchSystems.abstractBatchSystem.InsufficientSystemResources: Requesting more disk than either physically available, or enforced by --maxDisk. Requested: 53696612170, Available: 33709535232

I can't solve this issue  enforced by --maxDisk. and I did not understand the meaning of this message.
Please check my issue.

Thanks in advance!

**Comment by jvivian** [comment]
Hi @hmkim ,

The last line states the issue
```
Requesting more disk than either physically available, or enforced by --maxDisk. Requested: 53696612170, Available: 33709535232
```

The toil-rnaseq workflow is requesting ~53G of disk space and the location your work directory (which you can set via `--workDir`) is pointing to a location with only ~33GB.

**Comment by hmkim** [comment]
Thank you ! 

I solved this issue by setting via --workDir.
